pyvista/core/utilities/features.py

```diff
@@ -99,6 +99,120 @@ def voxelize(mesh, density=None, check_surface=True):
     return vox
 
 
+def voxelize_volume(mesh, density=None, check_surface=True):
+    """Voxelize mesh to create a RectilinearGrid voxel volume.
+
+    Creates a voxel volume that encloses the input mesh and discretizes the cells
+    within the volume that intersect and are contained within the input mesh.
+    All discretized cells are named ``MeshCells``. All other cells are named ``BackgroundCells``.
+
+    Parameters
+    ----------
+    mesh : pyvista.DataSet
+        Mesh to voxelize.
+
+    density : float | array_like[float]
+        The uniform size of the voxels when single float passed.
+        Nonuniform voxel size if a list of values are passed along x,y,z directions.
+        Defaults to 1/100th of the mesh length.
+
+    check_surface : bool, default: True
+        Specify whether to check the surface for closure. If on, then the
+        algorithm first checks to see if the surface is closed and
+        manifold. If the surface is not closed and manifold, a runtime
+        error is raised.
+
+    Returns
+    -------
+    pyvista.RectilinearGrid
+        RectilinearGrid as voxelized volume with discretized cells.
+
```

Comment:
We should add a see also link to the voxelize.  Not sure if my suggestion has a whitespace error since GitHub is finicky about multiline suggestions.
```suggestion
        RectilinearGrid as voxelized volume with discretized cells.

    See Also
    --------
    pyvista.voxelize

```